packages/spacecat-shared-rum-api-client/src/functions/opportunities/high-inorganic-high-bounce-rate.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import trafficAcquisition from '../traffic-acquisition.js';
+import { getCTRByUrl } from '../../common/aggregateFns.js';
+
+function convertToOpportunity(traffic) {
+  const { url, total, bounceRate } = traffic;
+
+  return {
+    type: 'high-inorganic-high-bounce-rate',
+    page: url,
+    screenshot: '',
+    trackedPageKPIName: 'Bounce Rate',
+    trackedPageKPIValue: bounceRate,
+    pageViews: total,
+    samples: total, // todo: get the actual number of samples
+    metrics: [], // none needed
```

Comment:
Can we provide, how much of inorganic traffic is coming and how much of them are engaging, in the metrics?